ui/src/views/404/index.jsx

```diff
@@ -1,5 +1,52 @@
 // @flow
+import { colour } from 'pasteup';
+import Logo from 'assets/images/guardian-logo-320.svg';
+import Navigation from './Navigation';
+import Footer from './Footer';
 
-import style from './style.js.scss';
+import {
+    heading,
+    fluidWrap,
+    topbar,
+    subHeadingContainer,
+    subHeading,
+    para,
+} from './style.js.scss';
 
-export default () => <div style={style.message}>404</div>;
+export default () =>
+    <div style={fluidWrap}>
+        <div style={topbar}>
+            <a
+                href="/"
+                style={{
+                    color: '#005689',
+                    textDecoration: 'none',
+                    fontFamily: 'arial',
+                    padding: '8px 10px 7px',
+                    display: 'block',
+                }}>
+                Home
+            </a>
+        </div>
+        <Logo
```

Comment:
@sndrs 
>  the loader could indeed generate a raster fallback
We wouldn't necessarily want this for every svg though

@SiAdcock 
> The underlying technology that comprises the component should, ideally, be of no concern to the developer who is using it

When you say this do you mean  html/css/js?

If so, you do need to be aware as a developer about what you are importing on the page. You should consider whether its the right component for you in the context you are creating. I feel like a danger of abstracting too much is:
* Promotes building using "blocks" you don't understand (I don't really feel like this is a good outcome)
* If you want to try to understand it becomes really difficult